Backend/app/dashboard/CommonAPIs/spec/RiderPlatform/AppManagement/API/Tickets.yaml

```diff
@@ -0,0 +1,91 @@
+imports:
+  APISuccess: Kernel.Types.APISuccess
+  TicketServiceResp: API.Types.UI.TicketService
+  TicketBookingUpdateSeatsReq: API.Types.UI.TicketService
+  TicketBookingCancelReq: API.Types.UI.TicketService
+  TicketServiceCancelReq: API.Types.UI.TicketService
+  TicketBookingDetails: API.Types.UI.TicketService
+  TicketServiceVerificationResp: API.Types.UI.TicketService
+  TicketService: Domain.Types.TicketService
+  TicketBookingService: Domain.Types.TicketBookingService
+  TicketPlace: Domain.Types.TicketPlace
+  TicketBooking: Domain.Types.TicketBooking
+
+importPackageOverrides:
+  API.Types.UI.TicketService: "rider-app"
+  Domain.Types.TicketService: "rider-app"
+  Domain.Types.TicketBookingService: "rider-app"
+  Domain.Types.TicketPlace: "rider-app"
+  Domain.Types.TicketBooking: "rider-app"
+
+module: Tickets
+apiPrefix: ""
+
+apis:
+  - POST: # VerifyBookingDetailsAPI
+      endpoint: /tickets/{personServiceId}/{ticketBookingShortId}/verify
+      auth: ApiAuthV2
+      params:
+        personServiceId: Id TicketService
+        ticketBookingShortId: ShortId TicketBookingService
+      response:
+        type: TicketServiceVerificationResp
+      migrate:
+        endpoint: TicketsAPI VerifyBookingDetails
+        userActionType: ApiAuth APP_BACKEND_MANAGEMENT CUSTOMERS VERIFY_BOOKING_DETAILS
+  - POST: # GetServicesAPI
+      endpoint: /{ticketPlaceId}/services
+      auth: ApiAuthV2
+      params:
+        ticketPlaceId: Id TicketPlace
+      query:
+        - date: Day
+      response:
+        type: "[TicketServiceResp]"
+      migrate:
+        userActionType: ApiAuth APP_BACKEND_MANAGEMENT CUSTOMERS GET_TICKET_SERVICES
+  - GET: # GetTicketPlacesAPI
+      endpoint:  /places
+      auth: ApiAuthV2
+      response:
+        type: "[TicketPlace]"
+      migrate:
+        userActionType: ApiAuth APP_BACKEND_MANAGEMENT CUSTOMERS GET_TICKET_PLACES
+  - POST: # UpdateSeatManagementAPI
+      endpoint: /update
```

Comment:
should we migrate paths for these apis, like `tickets/update`, `tickets/bookings/cancel`?